lib/howl/buffer.moon

```diff
@@ -295,9 +305,15 @@ class Buffer extends PropertyObject
     @viewers -= 1
 
   _associate_with_file: (file) =>
+    scope = @_config_scope
     @_file = file
+    config.copy scope, @_config_scope
```

Comment:
Given that `copy` really isn't a copy but an aliasing op, it seems like this would just have the effect of clobbering a scope? E.g. if a new buffer `b` has the config scope "buffer/1234", and is associated with file "/tmp/foo.moon", this would just overwrite any file scope with the buffer scope?